src/EventListener/InsertTagListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Terminal42\InsertTagsBundle\EventListener;
+
+use Contao\CoreBundle\DependencyInjection\Attribute\AsHook;
+use Psr\Log\LoggerInterface;
+use Terminal42\InsertTagsBundle\InsertTagHandler;
+
+class InsertTagListener
```

Comment:
```suggestion
class InsertTagsListener
```

same as the namespace and tag name.